### PR TITLE
Edge sends empty performance metrics to telemetry-analysis

### PR DESCRIPTION
**Pull request by edwardg8**

### CODE REVIEW

File: d2l-telemetry.js [hunk 1]
```diff
@@ -20,7 +20,13 @@
 			}
 			var perfObj = {};
 			performance.forEach(function(perf) {
-				perfObj[perf.name] = perf;
+				var perfMetricObj = {};
+				for (var key in perf) {
```
Comment (awikkerink) [inline d2l-telemetry.js]: Tastes awful, but it works.

Could you add a comment here explaining the purpose of this?

Comment (edwardg8) [inline d2l-telemetry.js]: sure

File: d2l-telemetry.js [hunk 2]
```diff
@@ -20,7 +20,15 @@
 			}
 			var perfObj = {};
 			performance.forEach(function(perf) {
-				perfObj[perf.name] = perf;
+				//Edge will fail to JSON.stringify(perf) directly, will return emptry string "{}"
```
Comment (awikkerink) [inline d2l-telemetry.js]: *nit* `emptry`?

Comment (awikkerink) [inline d2l-telemetry.js]: "{}" is not technically the empty **string**. It is an empty **object**

Comment (edwardg8) [inline d2l-telemetry.js]: Dah, changing it now

Comment (edwardg8) [inline d2l-telemetry.js]: what about `empty object string "{}"`, because json.stringfy returns string

**Comment by awikkerink** [comment]
LGTM :shipit:

**Pull request merged**